### PR TITLE
Continuation of BXMSDOC-8518 - remove jbpm.start.org from product docs

### PR DESCRIPTION
**Pull request by emmurphy1**
Continuing work from https://github.com/emmurphy1/kie-docs/pull/363.

PAM HTML draft:

http://file.ork.redhat.com/~emmurphy/BXMSDOC-8518-pam/#assembly-springboot-business-apps

DM HTML draft:

http://file.ork.redhat.com/~emmurphy/BXMSDOC-8518-dm/#assembly-springboot-business-apps

### CODE REVIEW

File: doc-content/enterprise-only/springboot/bus-app_con.adoc [hunk 1]
```diff
@@ -3,9 +3,7 @@
...
-
-On the https://start.jbpm.org[business application] website you can create a Process Automation Manager, Decision Manager, or {PLANNER} business application. After you create and configure your business application, you can deploy it to an existing service or to the cloud, through OpenShift.
+{PRODUCT} Spring Boot business applications are flexible, UI-agnostic logical groupings of individual services that provide certain business capabilities. Business applications are based on Spring Boot starters. They are usually deployed separately and can be versioned individually. A complete business application enables a domain to achieve specific business goals, for example order management or accommodation management. After you create and configure your business application, you can deploy it to an existing service or to the cloud, through OpenShift.
```
Comment (kaldesai) [inline doc-content/enterprise-only/springboot/bus-app_con.adoc]:
```suggestion
{PRODUCT} Spring Boot business applications are flexible, UI-agnostic logical groupings of individual services that provide certain business capabilities. Business applications are based on Spring Boot starters. They are usually deployed separately and can be versioned individually. A complete business application enables a domain to achieve specific business goals, for example, order management or accommodation management. After you create and configure your business application, you can deploy it to an existing service or to the cloud, through OpenShift.
```

**Review by kaldesai — APPROVED** [review]
@emmurphy1, I have verified your content and your changes look good. A minor suggestion about missing comma.

File: doc-content/enterprise-only/springboot/bus-app-async-execution-proc.adoc [hunk 2]
```diff
@@ -4,10 +4,10 @@
 After you create your Spring Boot project, you can use the `jbpm.executor` parameters to enable asynchronous execution.
 
 .Prerequisites
-* You have a Spring Boot business application ZIP file that you created using the http://start.jbpm.org[business application]  website.
+* You have a Spring Boot business application service file that you created using the Maven archetype command. For information, see xref:bus-app-create_{context}[].
```
Comment (kaldesai) [inline doc-content/enterprise-only/springboot/bus-app-async-execution-proc.adoc]:
```suggestion
* You have a Spring Boot business application service file that you created using the Maven archetype command. For more information, see xref:bus-app-create_{context}[].
```

File: doc-content/enterprise-only/springboot/bus-app-con-app-prop-proc.adoc [hunk 3]
```diff
@@ -4,10 +4,10 @@
 After you create your Spring Boot project, you can configure several components through the `application.properties` file to customize your project.
 
 .Prerequisites
-* You have a Spring Boot business application ZIP file that you created using the http://start.jbpm.org[business application]  website.
+* You have a Spring Boot business application service file that you created using the Maven archetype command. For information, see xref:bus-app-create_{context}[].
```
Comment (kaldesai) [inline doc-content/enterprise-only/springboot/bus-app-con-app-prop-proc.adoc]:
```suggestion
* You have a Spring Boot business application service file that you created using the Maven archetype command. For more information, see xref:bus-app-create_{context}[].
```

File: doc-content/enterprise-only/springboot/bus-app-con-rest-proc.adoc [hunk 4]
```diff
@@ -4,10 +4,10 @@
 After you create your Spring Boot project, you can configure  the host, port, and path for the REST endpoint for your Spring Boot application.
 
 .Prerequisites
-* You have a ZIP file that you created using the http://start.jbpm.org[business application]  website.
+* You have a Spring Boot business application service file that you created using the Maven archetype command. For information, see xref:bus-app-create_{context}[].
```
Comment (kaldesai) [inline doc-content/enterprise-only/springboot/bus-app-con-rest-proc.adoc]:
```suggestion
* You have a Spring Boot business application service file that you created using the Maven archetype command. For more information, see xref:bus-app-create_{context}[].
```

File: doc-content/enterprise-only/springboot/bus-app-config-data-source_proc.adoc [hunk 5]
```diff
@@ -4,10 +4,10 @@
 {PRODUCT} business applications are generated with the default H2 database. You can change the database type to MySQL or PostgreSQL.
 
 .Prerequisites
-* You have a Spring Boot application  file that you created using the http://start.jbpm.org[business applications] website.
+* You have a Spring Boot business application service file that you created using the Maven archetype command. For information, see xref:bus-app-create_{context}[].
```
Comment (kaldesai) [inline doc-content/enterprise-only/springboot/bus-app-config-data-source_proc.adoc]:
```suggestion
* You have a Spring Boot business application service file that you created using the Maven archetype command. For more information, see xref:bus-app-create_{context}[].
```

File: doc-content/enterprise-only/springboot/bus-app-config-jpa_proc.adoc [hunk 6]
```diff
@@ -4,10 +4,10 @@
 The Java Persistence API (JPA) is a standard technology that enables you to map objects to relational databases. You must configure JPA for your {PRODUCT} business application.
 
 .Prerequisites
-* You have a ZIP file that you created using the http://start.jbpm.org[business applications] website.
+* You have a Spring Boot business application service file that you created using the Maven archetype command. For information, see xref:bus-app-create_{context}[].
```
Comment (kaldesai) [inline doc-content/enterprise-only/springboot/bus-app-config-jpa_proc.adoc]:
```suggestion
* You have a Spring Boot business application service file that you created using the Maven archetype command. For more information, see xref:bus-app-create_{context}[].
```

File: doc-content/enterprise-only/springboot/bus-app-config-kie-server-proc.adoc [hunk 7]
```diff
@@ -4,10 +4,10 @@
 After you create your Spring Boot project, you can configure {KIE_SERVER} so that it can be easily identified.
 
 .Prerequisites
-* You have a Spring Boot business application ZIP file that you created using the http://start.jbpm.org[business application]  website.
+* You have a Spring Boot business application service file that you created using the Maven archetype command. For information, see xref:bus-app-create_{context}[].
```
Comment (kaldesai) [inline doc-content/enterprise-only/springboot/bus-app-config-kie-server-proc.adoc]:
```suggestion
* You have a Spring Boot business application service file that you created using the Maven archetype command. For more information, see xref:bus-app-create_{context}[].
```

File: doc-content/enterprise-only/springboot/bus-app-configure-central-proc.adoc [hunk 8]
```diff
@@ -4,10 +4,10 @@
 If you selected *Business Automation* when you created your Spring Boot business application, you can specify which {KIE_SERVER} components must start at runtime.
 
 .Prerequisites
-* You have a Spring Boot business application ZIP file that you created using the http://start.jbpm.org[business application]  website.
+* You have a Spring Boot business application service file that you created using the Maven archetype command. For information, see xref:bus-app-create_{context}[].
```
Comment (kaldesai) [inline doc-content/enterprise-only/springboot/bus-app-configure-central-proc.adoc]:
```suggestion
* You have a Spring Boot business application service file that you created using the Maven archetype command. For more information, see xref:bus-app-create_{context}[].
```

File: doc-content/enterprise-only/springboot/bus-app-quartz_proc.adoc [hunk 9]
```diff
@@ -4,7 +4,7 @@
 If you plan to run your application in a cluster, you must configure the Quartz timer service.
 
 .Prerequisites
-* You have a business application that you created using the http://start.jbpm.org[business application] website and you want to use this application in a cluster.
+* You have a Spring Boot business application service file that you created using the Maven archetype command. For information, see xref:bus-app-create_{context}[].
```
Comment (kaldesai) [inline doc-content/enterprise-only/springboot/bus-app-quartz_proc.adoc]:
```suggestion
* You have a Spring Boot business application service file that you created using the Maven archetype command. For more information, see xref:bus-app-create_{context}[].
```

File: doc-content/enterprise-only/springboot/bus-app-security-con.adoc [hunk 10]
```diff
@@ -12,7 +12,8 @@ A typical Spring Security implementation consists of the following steps that us
...
 
-Spring breaks these steps down into individual filters and chains them together in a FilterChain. This chaining method provides the flexibility required to work with almost any identity provider and security framework. With Spring Security, you can define a FilterChain for your application programmatically. The following section is from the `business-application-service/src/main/java/com/company/service/DefaultWebSecurityConfig.java` file generated as part of a business application created on the `https://start.jbpm.org` web site.
+Spring breaks these steps down into individual filters and chains them together in a FilterChain. This chaining method provides the flexibility required to work with almost any identity provider and security framework. With Spring Security, you can define a FilterChain for your application programmatically. The following section is from the `business-application-service/src/main/java/com/company/service/DefaultWebSecurityConfig.java` file from a Spring Boot business application service file created using the Maven archetype command. For information, see xref:bus-app-create_{context}[].
```
Comment (kaldesai) [inline doc-content/enterprise-only/springboot/bus-app-security-con.adoc:15]:
```suggestion
Spring breaks these steps down into individual filters and chains them together in a FilterChain. This chaining method provides the flexibility required to work with almost any identity provider and security framework. With Spring Security, you can define a FilterChain for your application programmatically. The following section is from the `business-application-service/src/main/java/com/company/service/DefaultWebSecurityConfig.java` file from a Spring Boot business application service file created using the Maven archetype command. For more information, see xref:bus-app-create_{context}[].
```

File: doc-content/enterprise-only/springboot/online-maven-configure-proc.adoc [hunk 11]
```diff
@@ -3,7 +3,7 @@
 After you create your {PRODUCT} Spring Boot project, configure it with the online Maven Repository to store your application data.
 
 .Prerequisites
-* You have created a {PRODUCT} Spring Boot project.
+* You have a Spring Boot business application service file that you created using the Maven archetype command. For information, see xref:bus-app-create_{context}[].
```
Comment (kaldesai) [inline doc-content/enterprise-only/springboot/online-maven-configure-proc.adoc]:
```suggestion
* You have a Spring Boot business application service file that you created using the Maven archetype command. For more information, see xref:bus-app-create_{context}[].
```

File: doc-content/enterprise-only/springboot/bus-app-create_proc.adoc [hunk 12]
```diff
@@ -1,36 +1,52 @@
...
-+
-* *Business Assets*: Contains business processes, rules, and forms and are easily imported into {CENTRAL}.  Select *Dynamic Assets* instead if you want to add adaptive and dynamic assets such as cases.
-* *Data Model*: Provides common data structures that are shared between the service projects and business assets projects. This enables proper encapsulation, promotes reuse, and reduces shortcuts. Each service project can expose its own public data model.
-* *Service*: Includes business logic that operates your business.
-. Click *Generate business application*.
-+
-The `<BUSINESS-APPLICATION>.zip` file downloads, where `<BUSINESS-APPLICATION>` is the name that you entered in the *Application Name* box.
-. Extract the `<BUSINESS-APPLICATION>.zip` file.
+* To create a service project, a deployable project that provides a service with various capabilities including the business logic that operates your business, enter one of the following commands:
+** To create a business application service project (the default configuration) that includes features for process management, case management, decision management, and optimisation:
```
Comment (gmunozfe) [inline doc-content/enterprise-only/springboot/bus-app-create_proc.adoc]: Not sure if optimization (with z) is more correct or they are interchangeable.

File: doc-content/enterprise-only/springboot/bus-app-create_proc.adoc [hunk 13]
```diff
@@ -1,36 +1,52 @@
...
++
+[source,subs="attributes+"]
+----
+mvn archetype:generate -B -DarchetypeGroupId=org.kie -DarchetypeArtifactId=kie-service-spring-boot-archetype -DarchetypeVersion={MAVEN_ARTIFACT_VERSION} -DgroupId=com.company -DartifactId=-service -Dversion=1.0-SNAPSHOT -Dpackage=com.company.service -DappType=brm
+----
+** To create a {PLANNER} service project to help you solve planning problems and solutions related features:
++
+[source,subs="attributes+"]
+----
+mvn archetype:generate -B -DarchetypeGroupId=org.kie -DarchetypeArtifactId=kie-service-spring-boot-archetype -DarchetypeVersion={MAVEN_ARTIFACT_VERSION} -DgroupId=com.company -DartifactId=-service -Dversion=1.0-SNAPSHOT -Dpackage=com.company.service -DappType=planner
```
Comment (gmunozfe) [inline doc-content/enterprise-only/springboot/bus-app-create_proc.adoc]: Missing <project> at artifactId:
```suggestion
mvn archetype:generate -B -DarchetypeGroupId=org.kie -DarchetypeArtifactId=kie-service-spring-boot-archetype -DarchetypeVersion={MAVEN_ARTIFACT_VERSION} -DgroupId=com.company -DartifactId=<project>-service -Dversion=1.0-SNAPSHOT -Dpackage=com.company.service -DappType=planner
```

File: doc-content/enterprise-only/springboot/bus-app-create_proc.adoc [hunk 14]
```diff
@@ -1,36 +1,52 @@
...
++
+[source,subs="attributes+"]
+----
+mvn archetype:generate -B -DarchetypeGroupId=org.kie -DarchetypeArtifactId=kie-service-spring-boot-archetype -DarchetypeVersion={MAVEN_ARTIFACT_VERSION} -DgroupId=com.company -DartifactId=<project>-service -Dversion=1.0-SNAPSHOT -Dpackage=com.company.service -DappType=bpm
+----
+** To create a decision management service project that includes decision and rules-related features:
++
+[source,subs="attributes+"]
+----
+mvn archetype:generate -B -DarchetypeGroupId=org.kie -DarchetypeArtifactId=kie-service-spring-boot-archetype -DarchetypeVersion={MAVEN_ARTIFACT_VERSION} -DgroupId=com.company -DartifactId=-service -Dversion=1.0-SNAPSHOT -Dpackage=com.company.service -DappType=brm
```
Comment (gmunozfe) [inline doc-content/enterprise-only/springboot/bus-app-create_proc.adoc]: Missing <project> at artifactId:
```suggestion
mvn archetype:generate -B -DarchetypeGroupId=org.kie -DarchetypeArtifactId=kie-service-spring-boot-archetype -DarchetypeVersion={MAVEN_ARTIFACT_VERSION} -DgroupId=com.company -DartifactId=<project>-service -Dversion=1.0-SNAPSHOT -Dpackage=com.company.service -DappType=brm
```

**Review by gmunozfe — COMMENTED** [review]
Looks good to me, good work @emmurphy1 !
Just a few minor comments

File: doc-content/enterprise-only/springboot/bus-app_con.adoc (hunk 1 above)
Comment (emmurphy1) [reply doc-content/enterprise-only/springboot/bus-app_con.adoc]: Good catch, @kaldesai !

File: doc-content/enterprise-only/springboot/bus-app-create_proc.adoc (hunk 12 above)
Comment (emmurphy1) [reply doc-content/enterprise-only/springboot/bus-app-create_proc.adoc]: Good catch, @gmunozfe. We use American English so should have the z.

**Review by gmunozfe — APPROVED** [review]
Looks good to me, great work @emmurphy1 !

**Pull request merged**